client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -49,20 +52,26 @@ export const Admin = () => {
     }
   }, [isLoggedIn, token, isAuthenticating])
 
+  // Accesses the database and fetches all reviews. Called when admin page loads, and
+  // splits the reviews into three categories: approved (visible on the website),
+  // pending (awaiting approval), and reported (hidden and awaiting approval)
   useEffect(() => {
     axios
-      .post('/api/fetchPendingReviews', { token: token })
+      .post('/api/fetchAllReviews', { token: token })
       .then((response) => {
         const result = response.data.result
         if (response.status === 200) {
-          setUnapprovedReviews(
-            result.filter((review: Review) => review.reported === 0)
+          setApprovedReviews(
+            result.filter((review: Review) => review.visible === 1)
+          )
+          setPendingReviews(
+            result.filter((review: Review) => review.reported === 0 && review.visible === 0)
           )
           setReportedReviews(
-            result.filter((review: Review) => review.reported === 1)
+            result.filter((review: Review) => review.reported === 1 && review.visible === 0)
           )
         } else {
-          console.log('Error at fetchPendingReviews')
+          console.log('Error at fetchAllReviews')
```

Comment:
Looks like there are console logs initially - (from before you worked on it) - fine for testing right now but should delete all console logs when you are done